docs/git-bundles.md

```diff
@@ -94,3 +91,42 @@ spec:
       repository: https://github.com/exdx/combo-bundle
   provisionerClassName: core.rukpak.io/plain
 ```
+
+## Private git repositories
+
+A Bundle can reference content in a private git repository using HTTPS protocol by creating a secret in the namespace that the provisioner is deployed.
+The secret is expected to contain `data.username` and `data.accesstoken` for the username and personal access token, respectively.
+The personal access token can be generated in the github settings.  ("Setting" -> "Developer settings" -> "Personal access tokens")
```

Comment:
Would it make sense to move this header above the Examples header? Thoughts?

Reply:
Also should we link out to https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/creating-a-personal-access-token?

Reply:
It depends how many people want to set the credential. It probably helps the rate limit issues avoidance as well as the private repository access.

Reply:
I'll add the link.  Thanks!

Reply:
I'll move this section up.